pages/news.html

```diff
@@ -88,6 +88,7 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
+            <p><strong>05/04/2023</strong> - We started a <a href="https://github.com/uyuni-project/uyuni/discussions/6821" target="_blankl">poll</a> to decide if we to switch async discussions from <a href="../pages/contact.html#ml">Mailing lists</a> to <a href="https://github.com/uyuni-project/uyuni/discussions" target="_blank">GitHub conversations</a>.</p>
```

Comment:
```suggestion
            <p><strong>05/04/2023</strong> - We started a <a href="https://github.com/uyuni-project/uyuni/discussions/6821" target="_blankl">poll</a> to decide if we want to switch async discussions from <a href="../pages/contact.html#ml">mailing lists</a> to <a href="https://github.com/uyuni-project/uyuni/discussions" target="_blank">GitHub conversations</a>.</p>
```

Reply:
In fact I wanted to avoid "want to". So will remove that.